libs/core/src/lib/splitter/splitter.component.scss

```diff
@@ -0,0 +1,227 @@
+        flex-grow: 1;
+        overflow: hidden;
+
+        &-wrapper {
+            flex-grow: 1;
+            display: flex;
+            flex-direction: column;
+        }
+
+        &--root {
+            height: calc(100% - 2.7rem);
+        }
+
+        &--horizontal {
+            flex-direction: column;
+
+            & > .#{$block}__resizer {
+                width: 100%;
+                height: 1rem;
+                cursor: row-resize;
+
+                &.#{$block}__resizer:hover,
+                &.#{$block}__resizer--active {
+                    .#{$block}__resizer-decoration-before,
+                    .#{$block}__resizer-decoration-after {
+                        width: 100%;
+                    }
+                }
+
+                .#{$block}__resizer-decoration-before,
+                .#{$block}__resizer-decoration-after {
+                    width: 4rem;
+                    height: 0.0625rem;
+                }
+
+                .#{$block}__resizer-decoration-before {
+                    @include fd-gradient(90deg);
+                }
+
+                .#{$block}__resizer-decoration-after {
+                    @include fd-gradient(270deg);
+                }
+
+                .#{$block}__resizer-grip {
+                    width: 2rem;
+                    height: 1.5rem;
+                }
+            }
+        }
+
+        &--vertical {
+            flex-direction: row;
+
+            & > .#{$block}__resizer {
+                height: 100%;
+                width: 1rem;
+                cursor: col-resize;
+                flex-direction: column;
+
+                &.#{$block}__resizer:hover,
+                &.#{$block}__resizer--active {
+                    .#{$block}__resizer-decoration-before,
+                    .#{$block}__resizer-decoration-after {
+                        height: 100%;
+                    }
+                }
+
+                .#{$block}__resizer-decoration-before,
+                .#{$block}__resizer-decoration-after {
+                    width: 0.0625rem;
+                    height: 4rem;
+                }
+
+                .#{$block}__resizer-decoration-before {
+                    @include fd-gradient(180deg);
+                }
+
+                .#{$block}__resizer-decoration-after {
+                    @include fd-gradient(0deg);
+                }
+
+                .#{$block}__resizer-grip {
+                    width: 1.5rem;
+                    height: 2rem;
+                }
+            }
+        }
+    }
+
+    &__split-pane {
+        flex-grow: 1;
+        overflow: auto;
+        text-overflow: ellipsis;
+    }
+
+    &__resizer {
+        background-color: #edeff0;
+        background-color: var(--sapUiShellBackground, #edeff0);
+        outline: none;
+
+        @include fd-flex-center();
+
+        .#{$block}__resizer-decoration-before,
+        .#{$block}__resizer-decoration-after {
+            transition: all 0.1s ease-in;
+        }
```

Comment:
This block of code is repeated many times. It can go to a mixin:

```
@mixin fd-splitter-some-name() {
     .#{$block}__resizer-decoration-before,
     .#{$block}__resizer-decoration-after {
             @content;
     }
}
```